test/index.js

```diff
@@ -634,3 +634,16 @@ test('is.symbol', function (t) {
 
   t.end();
 });
+
+// no need to exhaustively test all negated methods. just need to test structure.
```

Comment:
please do exhaustively test all negated methods - the tests shouldn't know that the implementation is creating them in an automated fashion.